vcs.go

```diff
@@ -50,128 +33,127 @@ func createDevelDB() error {
 	return err
 }
 
-// parseSource returns owner and repo from source
-func parseSource(source string) (owner string, repo string) {
+// parseSource returns the git url and efault branch
```

Comment:
*default branch

just a typo

Reply:
I made that comment before i added the protocols support.

Fixed the typo and mentioned protocols.